src/common/VersionView/VersionCheckbox/VersionCheckbox.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+import { Checkbox } from '@folio/stripes/components';
+
+export const VersionCheckbox = ({
+  checked,
+  marked,
+  label,
+  ...props
+}) => {
+  const checkboxLabel = marked ? <mark>{label}</mark> : label;
+
+  return (
+    <Checkbox
+      checked={Boolean(checked)}
+      disabled
+      label={checkboxLabel}
+      vertical
+      {...props}
+    />
+  );
+};
+
+VersionCheckbox.propTypes = {
+  checked: PropTypes.bool,
```

Comment:
Could you please investigate can we add default value for not required attribute across this pull request?

Reply:
Add default values for `VersionCheckbox` and `VersionKeyValue` boolean props, the rest is ok as `undefined`